recipes/libnsl/build.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
+
+./configure --prefix=${PREFIX} --disable-static
+make
```

Comment:
Is there a `make check` or `make test`?

Reply:
Yes, though I don't think it does much. Added in https://github.com/conda-forge/staged-recipes/pull/16278/commits/06c973e02e0e11383fe05e19a5c87c7ba6382d1b